pkg/ingester/ingester.go

```diff
@@ -85,6 +85,7 @@ const (
 	instanceIngestionRateTickInterval = time.Second
 
 	sampleOutOfOrder     = "sample-out-of-order"
+	sampleTooOld         = "sample-too-old"
```

Comment:
I think we're going to have some confusion between `sampleOutOfBounds` and `sampleTooOld`. From an user perspective, they're the same: the same timestamp is too old. What if we merge them?

Reply:
The other error is sample-out-of-bounds or sample-out-of-order. We intentionally changed the error message to have a single on when out of order support is enabled: it is either too old or not too old. What do you suggest here? (the other case was simple where you called out of bound sample as too old)

Reply:
I see what you mean. Practically, isn't always an "out of bounds" error? Even when you enable the "out of order ingestion" you still have "bounds". I would like to hear @pstibrany opinion too, cause I also see a value in keeping the different cases distinct in metrics. 🤔

Reply:
I would start with separate metrics so that we can learn more about the feature, but I don't have a strong opinion about it.

Reply:
Yes, we still have bounds. But the meaning before and after ooo is totally different. "too old" felt more natural. Maybe re-using out-of-bounds is a smarter idea (not too late to make that change btw).